README.md

```diff
@@ -278,6 +278,12 @@ accepting connections
 ### Running under non-root
 By default, the main Manticore process `searchd` is running under user `manticore` inside the container, but the script which runs on starting the container is run under your default docker user which in most cases is `root`. If that's not what you want you can use `docker ... --user manticore` or `user: manticore` in docker compose yaml to make everything run under `manticore`. Read below about possible volume permissions issue you can get and how to solve it.
 
+### Creating plain indexes on startup
+Manticore doesn't create newly added plain indexes on its own startup (you need to run indexer). If you want to create them on startup you can use `CREATE_PLAIN_INDEXES=1` environment variable. It will run `indexer --all` before Manticore start. It is useful if you don't use volumes and your indexes are easy to recreate.
```

Comment:
```suggestion
Manticore doesn't create newly added plain indexes on its own startup, you need to run the indexer. To create them on startup, you can use the CREATE_PLAIN_INDEXES=1 environment variable. It will execute indexer --all before Manticore starts. This is useful if you don't use volumes, and your indexes are easy to recreate.
```